src/service/countries.service.ts

```diff
@@ -0,0 +1,10 @@
+export const getCountriesList = () => {
+  return fetch("https://restcountries.com/v2/all")
```

Comment:
ogólnie spoko, ale raz używasz then, a raz używasz async await (w innej funkcji), staraj się trzymać jednego sposobu

Reply:
konwencja z zapisem async await chyba jest krótsza. Warto wtedy zmienić ją na async await dla czytelnosci? Nie orientuje sie jeszcze jak wtedy obsluzyc bledy

Reply:
w async/await właśnie obsługa błędów jest przyjemniejsza i bardziej czytelna, ale to nieważne. Ważne, abyś skorzystał z jednej konwencji i się jej trzymał w projekcie.

Reply:
No to jak wiesz, że nie wiesz, jak obsłużyć błędy ze składnią async/await to warto sprawdzić, zamiast odkładać to na później